ansible/roles/keystone_bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Admin needs admin role for default domain
+  openstack.cloud.role_assignment:
+    user: "{{ lookup('ansible.builtin.env', 'OS_USERNAME', default=Undefined) }}"
+    domain: "{{ lookup('ansible.builtin.env', 'OS_DEFAULT_DOMAIN', default=Undefined) }}"
```

Comment:
this is using different defaults - are the `OS_USERNAME` and `OS_DEFAULT_DOMAIN` set by the container somewhere? In OSH, we had those set in the init container but here I think they will be missing

Reply:
They are being populated by OSH. The original script used these environment variables so I just continued that behavior.

Reply:
They're set in the job definition by OSH as environment variables.